pages.it/linux/i3.md

```diff
@@ -0,0 +1,37 @@
+# i3
+
+> Window Manager Dinamico.
+> Tasto Windows=Super=Mod4, Alt=Mod1
+> Maggiori Informazioni: <https://i3wm.org/docs/userguide.html>.
+
+- Avvia i3 (non può essere eseguito contemporaneamente con un altro gestore di finestre):
+
+`i3`
+
+- Apri un terminale:
+
+`Super + Invio`
+
+- Crea un nuovo workspace:
+
+`Super + Shift + {{numero}}`
+
+- Salta a workspace {{numero}}:
+
+`Super + {{numero}}`
+
+- Per il prossimo nuovo, dividi orizzontalmente:
+
+`Super + h`
+
+- Per il prossimo nuovo, dividi verticalmente:
+
+`Super + v`
+
+- Avvia dmenu, lanciatore di programmi:
+
+`Super + D`
+
+- Apri configurazione:
+
+`{{vim|nano}} ~/.i3/config`
```

Comment:
This is not an example about the `i3` command itself, and also we try to translate only examples in the main (English) page because it's easier to keep them in sync later.
:bulb: Tip: you can check outdated translations here: https://lukwebsforge.github.io/tldri18n/
```suggestion
```
If you think this is needed we can upgade the `i3` page when this translation is merged.